_common/qiskit/execute.py

```diff
@@ -365,77 +371,31 @@ def set_execution_target(backend_id='qasm_simulator',
                     
             backend.latest_session = session
             
-        ###############################
-        # if using IBM Quantum Platform, assume the backend_id is given only
-        elif use_ibm_quantum_platform: 
-        
-            from qiskit import IBMQ
-            if IBMQ.stored_account():
-            
-                # load a stored account
-                IBMQ.load_account()
-                
-                # set use_sessions in provided by user - NOTE: this will modify the global setting
-                this_use_sessions = exec_options.get("use_sessions", None)
-                if this_use_sessions != None:
-                    use_sessions = this_use_sessions
-
-                # if use sessions, setup runtime service, Session, and Sampler
-                if use_sessions:
-                    from qiskit_ibm_runtime import QiskitRuntimeService, Session, Options, SamplerV2 as Sampler
-                    
-                    service = QiskitRuntimeService()
-                    session_count += 1
-                    
-                    backend = service.backend(backend_id)
-                    session = Session(service=service, backend=backend_id)
-                    
-                    # get Sampler resilience level and transpiler optimization level from exec_options
-                    options = Options()
-                    # options.resilience_level = exec_options.get("resilience_level", 1)
-                    # options.optimization_level = exec_options.get("optimization_level", 3)
-                    
-                    # special handling for ibmq_qasm_simulator to set noise model
-                    if backend_id == "ibmq_qasm_simulator":
```

Comment:
The cloud simulators are retired as of May 2024; see the announcement [here](https://docs.quantum.ibm.com/announcements/product-updates/2024-05-15-2024-sunset-final-lab-simulators). For simulating circuits with custom noise model, one can use the local `Aer` simulators. The migration guide can be found [here](https://docs.quantum.ibm.com/migration-guides/local-simulators).